### DISCUSSION TITLE
Has anyone deployed Conductor on GCP (Google Cloud)?

### DISCUSSION THREAD

**Discussion by rickfish** [Netflix/conductor#2918] [General]
Has anyone deployed Conductor on GCP? I know that AWS is the preferred cloud provider and there are some integrations there. My current employer has decided on GCP as their cloud provider and wanted to get a head start on deploying Conductor there if possible...

**Comment by v1r3n** [comment]
Hi @rickfish we are working on the GCP deployments, happy to collaborate if you are interested.

**Comment by rickfish** [comment]
@v1r3n Definitely interested. I am new to this customer but they could use a workflow engine for one of their projects. I will be doing a POC on my local machine first but shortly thereafter if they agree that Conductor will be useful we will want to deploy it on GCP. Do you know what persistence module you will use and what you will use for queueing? My last customer used Postgres for both but this customer can use anything...

**Comment by jhonnyman** [comment]
Hello all I'm also very interested in deploying Conductor to GCP.
We are currently prototyping a solution to validate our use cases locally.

Our requirements would be:
- Persist workflow data on Postgres (GCP Cloud SQL)
- GCP Pub/Sub or GCP Task Queue for message queueing
- GCP Bucket for external storage (we mainly deal with files)

Would appreciate any guide on this topic.

**Reply by v1r3n** [reply]
@jhonnyman happy to chat, we are working to provide options to run Conductor on GCP.

**Comment by rakesh-k-dunzo** [comment]
Is this initiative being tracked anywhere?

I also have a similar use case. 

- GCP Cloud SQL (Postgres) for persistence
- Pub/Sub for Task Queues
- GKE for hosting Conductor servers
- Elastico for ES

**Reply by dilip-lukose** [reply]
Hi @rakesh-k-dunzo , Orkes provides fully managed Conductor clusters on GCP (two deployment options: deployed on customer's cloud account or Orkes'). Let me know if this maps to your needs and happy to chat more.

**Reply by rakesh-k-dunzo** [reply]
Thanks @dilip-lukose. I was also evaluating Orkes version of Conductor. Will checkout this option there.
